web-admin/src/features/projects/user-management/UserItem.svelte

```diff
@@ -2,40 +2,53 @@
   import AvatarListItem from "@rilldata/web-admin/features/organizations/users/AvatarListItem.svelte";
   import UserSetRole from "./UserSetRole.svelte";
   import { createAdminServiceGetCurrentUser } from "@rilldata/web-admin/client";
-  import type { V1ProjectMemberUser } from "@rilldata/web-admin/client";
-
-  type User =
-    | V1ProjectMemberUser
-    | {
-        userName: any;
-        userEmail: string;
-        userPhotoUrl?: string;
-        roleName: string;
-        email?: string;
-        role?: string;
-        invitedBy?: string;
-      };
+  import type {
+    V1ProjectMemberUser,
+    V1ProjectInvite,
+  } from "@rilldata/web-admin/client";
+
+  type User = V1ProjectMemberUser | V1ProjectInvite;
 
   export let organization: string;
   export let project: string;
   export let user: User;
+  export let orgRole: string;
+  export let manageProjectMembers: boolean;
+  export let manageProjectAdmins: boolean;
 
   $: currentUser = createAdminServiceGetCurrentUser();
+
+  $: showGuestChip = orgRole === "guest";
```

Comment:
Rather, can we use the `guest` prop from `V1OrganizationPermissions`? It could be passed through like the other permissions.

Reply:
i'm unsure if i should use the guest prop from org permissions based on [this comment](https://github.com/rilldata/rill/pull/7242#discussion_r2070583029).

since we now get the user's orgRole in the project members and project invites' API response, it is pretty accurate to use the user's orgRole to show the guest chip as a result.

Reply:
I see. Agreed– we should use the `orgRole` for rendering any role chips.